<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atrial fibrillation detection service validation tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDockWidget" name="controlDock">
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="processButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="plotButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Plot RR interval and analysis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ecgPlotButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Plot ECG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="labelAllAfRegionsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Label all AF regions.&lt;/p&gt;&lt;p&gt;A region will be created whenever the estibated AF probability is over the threshold. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Label all AF regions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resultAnalysisButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Result analysis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Threshold setting</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="threesholdLabel">
             <property name="text">
              <string>Threshold value = 0.5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="threshold">
             <property name="toolTip">
              <string>To update the Threshold value in the graphs, press the 'Plot Data' button. </string>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>RR interval signal display setting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QRadioButton" name="radioButtonLine">
             <property name="text">
              <string>Line (Fast display response)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonDot">
             <property name="text">
              <string>Dot (Slow display response)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Source file:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="loadFilePath">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="graphsDock">
      <property name="windowTitle">
       <string/>
      </property>
      <widget class="QWidget" name="dockWidgetContents_4">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PlotWidget" name="graphWidget4">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlotWidget" name="graphWidget1">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlotWidget" name="graphWidget2">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlotWidget" name="graphWidget3">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="configDock">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="markerTime">
          <property name="text">
           <string>Marker time:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="rrAfRegionTable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>210</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Start</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stop</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addAfButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add AF region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAfButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove selected AF region</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ecgRegions">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="ecgMarkerTime">
       <property name="text">
        <string>Marker time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="ecgAfRegionTable">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>210</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Start</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stop</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addEcgAfButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add AF region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEcgAfButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove selected AF region</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="inferencePerformance">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="cmView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>No confusion matrix</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="accuracyLabel">
       <property name="text">
        <string>Accuracy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sensitivityLabel">
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="specificityLabel">
       <property name="text">
        <string>Specificity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="f1Label">
       <property name="text">
        <string>F1-score</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="rocDock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="rocView">
          <property name="text">
           <string>No ROC</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="aucLabel">
       <property name="text">
        <string>AUC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
